scopes/envs/envs/environments.main.runtime.ts

```diff
@@ -359,6 +359,11 @@ export class EnvsMain {
     let envIdFromEnvsConfigWithoutVersion;
 
     if (envIdFromEnvsConfig) {
+      const envDefWithVersion = this.getEnvDefinitionByStringId(envIdFromEnvsConfig);
```

Comment:
This is not the right solution for the issue.
You should save it without version in the envs/envs when doing envs set command